src/components/organisms/OakHeader/OakHeader.tsx

```diff
@@ -0,0 +1,191 @@
+                $maxWidth={"all-spacing-22"}
+                $minHeight={[
+                  "all-spacing-0",
+                  "all-spacing-0",
+                  "all-spacing-20",
+                ]}
+                $mb={[
+                  "space-between-none",
+                  "space-between-l",
+                  "space-between-none",
+                ]}
+                $justifyContent={"center"}
+              >
+                <OakHeading
+                  tag="h2"
+                  $color={"grey80"}
+                  $font={["heading-4", "heading-3"]}
+                  $mb={"space-between-m"}
+                >
+                  {headingTitle}
+                </OakHeading>
+                <OakFlex
+                  $mb={"space-between-m"}
+                  $flexDirection={"row"}
+                  $alignItems={"center"}
+                >
+                  <StyledAutorImage
+                    alt={`${authorName} profile image`}
+                    src={authorImageSrc}
+                    $zIndex={"in-front"}
+                  />
+                  <OakBox>
+                    <OakHeading tag="h3" $font={"heading-7"}>
+                      {authorName}
+                    </OakHeading>
+                    <OakP>{authorTitle}</OakP>
+                  </OakBox>
+                </OakFlex>
+                <OakP $mb={["space-between-l", "space-between-none"]}>
+                  {subHeadingText}
+                </OakP>
+              </OakFlex>
+            </OakFlex>
+
+            <OakFlex
+              $position={"relative"}
+              $minHeight={[
+                "all-spacing-18",
+                "all-spacing-21",
+                "all-spacing-21",
+              ]}
+              $alignItems={"center"}
+              $justifyContent={"center"}
+              $mb={[
+                "space-between-none",
+                "space-between-m2",
+                "space-between-none",
+              ]}
+            >
+              <OakImage
+                $transform={"rotate(-2.025deg)"}
+                src={heroImageSrc}
+                width={100}
+                $minHeight={[
+                  "all-spacing-19",
+                  "all-spacing-20",
+                  "all-spacing-20",
+                ]}
+                $minWidth={[
+                  "all-spacing-19",
+                  "all-spacing-22",
+                  "all-spacing-21",
+                ]}
+                alt={`${headingTitle} hero image`}
+                $zIndex={"in-front"}
+              />
+              <OakFlex
+                $position={"absolute"}
+                $top={"all-spacing-0"}
+                $bottom={"all-spacing-0"}
+                $width={["all-spacing-20", "all-spacing-23", "all-spacing-23"]}
+              >
+                <OakImage
+                  src={
+                    "https://res.cloudinary.com/oak-web-application/image/upload/v1699887218/ui-graphics/loopdown.svg"
```

Comment:
can you add this svg to the image map and use `OakIcon` here? it constructs the URL using env vars instead of hardcoding it

Reply:
✅